Quantum-Control-Applications/ESR/input_calibration.py

```diff
@@ -0,0 +1,75 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+###################
+# The QUA program #
+###################
+
+with program() as input_cal:
+
+    n = declare(int)
+
+    adc_st = declare_stream(adc_trace=True)
+
+    with for_(n, 0, n < 5000, n + 1):
+        play("activate", "switch_receiver", duration=500)
+        wait(150, "resonator")
+        reset_phase("resonator")
```

Comment:
I assume that you want to measure the offsets after playing 'activate' on 'switch_receiver' because that's how you will measure data in the real sequences, right ?
Does the wait value matter here? because it seems to be different from what you used in the other sequences

Reply:
does not matter